### PR TITLE
placementinsql: add the survival preferences docs

### PR DESCRIPTION
**Pull request by nolouch**
Signed-off-by: nolouch <nolouch@gmail.com>

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

wait https://github.com/pingcap/tidb/pull/40613

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by SunRunAway — APPROVED** [review]

File: placement-rules-in-sql.md [hunk 1]
```diff
@@ -247,6 +248,25 @@ PARTITION BY RANGE( YEAR(purchased) ) (
...
+### 生存能力偏好设置
+
+一些重要的数据可能需要具备跨可用区存储多个副本，从而具备高的容灾生存能力，比如具有 Region 级别的生存能力，`SURVIVAL_PREFERENCES` 可以提供生存能力的偏好设置。
+以下示例设置了一个约束，要求数据尽量满足生存偏好设置：
+
+``` sql
+CREATE PLACEMENT POLICY multiregion
+    follower=4
+    PRIMARY_REGION="region1"
+    SURVIVAL_PREFERENCES="[region, zone]";
```
Comment (qiancai) [inline placement-rules-in-sql.md:261]: 这里的 SURVIVAL_PREFERENCES 可以设置成哪些值呢，只能设置为 "[region, zone]" 吗，还是可以设置成其他的呢

File: placement-rules-in-sql.md [hunk 2]
```diff
@@ -247,6 +248,25 @@ PARTITION BY RANGE( YEAR(purchased) ) (
...
+```
+
+设置了该策略的表，数据会先优先满足跨 Region 级别数据隔离的生存目标，再保证跨 zone 级别的数据隔离的生存目标。
```
Comment (qiancai) [inline placement-rules-in-sql.md:264]: “跨 Region 级别数据隔离”里的 Region 是指 region 吗，比如 us-east-1

File: placement-rules-in-sql.md [hunk 3]
```diff
@@ -247,6 +248,25 @@ PARTITION BY RANGE( YEAR(purchased) ) (
...
+
+### 生存能力偏好设置
+
+一些重要的数据可能需要具备跨可用区存储多个副本，从而具备高的容灾生存能力，比如具有 Region 级别的生存能力，`SURVIVAL_PREFERENCES` 可以提供生存能力的偏好设置。
+以下示例设置了一个约束，要求数据尽量满足生存偏好设置：
+
+``` sql
+CREATE PLACEMENT POLICY multiregion
+    follower=4
+    PRIMARY_REGION="region1"
```
Comment (qiancai) [inline placement-rules-in-sql.md:260]:
```suggestion
    PRIMARY_REGION="us-east-1"
```

File: placement-rules-in-sql.md [hunk 4]
```diff
@@ -247,6 +248,25 @@ PARTITION BY RANGE( YEAR(purchased) ) (
 >
 > 字典和列表格式都基于 YAML 解析，但 YAML 语法有些时候不能被正常解析。例如 YAML 会把 `"{+disk=ssd:1,+disk=nvme:2}"`（`:` 后无空格）错误地解析成 `'{"+disk=ssd:1": null, "+disk=nvme:2": null}'`，不符合预期。但 `"{+disk=ssd: 1,+disk=nvme: 2}"`（`:` 后有空格）能被正确解析成 `'{"+disk=ssd": 1, "+disk=nvme": 2}'`。
 
+
+### 生存能力偏好设置
+
+一些重要的数据可能需要具备跨可用区存储多个副本，从而具备高的容灾生存能力，比如具有 Region 级别的生存能力，`SURVIVAL_PREFERENCES` 可以提供生存能力的偏好设置。
+以下示例设置了一个约束，要求数据尽量满足生存偏好设置：
```
Comment (qiancai) [inline placement-rules-in-sql.md:255]:
```suggestion

以下示例设置了一个约束，要求数据尽量满足生存偏好设置：
```

File: placement-rules-in-sql.md [hunk 5]
```diff
@@ -247,6 +248,25 @@ PARTITION BY RANGE( YEAR(purchased) ) (
 >
 > 字典和列表格式都基于 YAML 解析，但 YAML 语法有些时候不能被正常解析。例如 YAML 会把 `"{+disk=ssd:1,+disk=nvme:2}"`（`:` 后无空格）错误地解析成 `'{"+disk=ssd:1": null, "+disk=nvme:2": null}'`，不符合预期。但 `"{+disk=ssd: 1,+disk=nvme: 2}"`（`:` 后有空格）能被正确解析成 `'{"+disk=ssd": 1, "+disk=nvme": 2}'`。
 
+
+### 生存能力偏好设置
```
Comment (SunRunAway) [inline placement-rules-in-sql.md:252]: 专有名词别翻译了吧。就叫 SURVIVAL_PREFERENCE 如何 @qiancai

File: placement-rules-in-sql.md [hunk 6]
```diff
@@ -247,6 +248,25 @@ PARTITION BY RANGE( YEAR(purchased) ) (
...
+
+一些重要的数据可能需要具备跨可用区存储多个副本，从而具备高的容灾生存能力，比如具有 Region 级别的生存能力，`SURVIVAL_PREFERENCES` 可以提供生存能力的偏好设置。
+以下示例设置了一个约束，要求数据尽量满足生存偏好设置：
+
+``` sql
+CREATE PLACEMENT POLICY multiregion
```
Comment (SunRunAway) [inline placement-rules-in-sql.md:258]: 建议写两个例子吧。把例子描述写多一些。
1. 多region部署的场景。
2. 不同表，有些表单az，有些表跨az高可用

File: placement-rules-in-sql.md (hunk 5 above)
Comment (SunRunAway) [inline placement-rules-in-sql.md:252]:
```suggestion
### 生存偏好
```

File: placement-rules-in-sql.md (hunk 1 above)
Comment (nolouch) [reply placement-rules-in-sql.md:262]: 可以设置成其他的

File: placement-rules-in-sql.md [hunk 7]
```diff
@@ -247,6 +248,26 @@ PARTITION BY RANGE( YEAR(purchased) ) (
...
+    follower=4
+    PRIMARY_REGION="region1"
+    SURVIVAL_PREFERENCES="[region, zone]";
+```
+
+设置了该策略的表，数据会先优先满足跨 Region 级别数据隔离的生存目标，再保证跨 zone 级别的数据隔离的生存目标。
```
Comment (qiancai) [inline placement-rules-in-sql.md:265]:
```suggestion
对于绑定了该放置策略的表，数据会先优先满足跨 `region` （即云区域）级别数据隔离的生存目标，再满足跨可用区域（即 zone）级别的数据隔离的生存目标。
```

File: placement-rules-in-sql.md (hunk 3 above)
Comment (qiancai) [reply placement-rules-in-sql.md:261]: 这篇文档里大部分地方 PRIMARY_REGION  的举例值都是 "us-east-1"

File: placement-rules-in-sql.md (hunk 6 above)
Comment (nolouch) [reply placement-rules-in-sql.md:259]: 这个已经是多 region 的例子了

> 有些表单az，有些表跨az高可用。

这理只阐述跨 az，和这个小章节一致

File: placement-rules-in-sql.md [hunk 8]
```diff
@@ -134,6 +134,7 @@ SELECT * FROM information_schema.partitions WHERE tidb_placement_policy_name IS
 | `FOLLOWER_CONSTRAINTS`     | 仅适用于 follower 的约束列表。                                           |
 | `LEARNER_CONSTRAINTS`     | 仅适用于 learner 的约束列表。                                           |
 | `LEARNERS`                 | 指定 learner 的数量。     |
+| `SURVIVAL_PREFERENCE`      | 指定按 label 容灾等级的优先级放置副本。例如 `SURVIVAL_PREFERENCE="[region, zone, host]"`    |
```
Comment (qiancai) [inline placement-rules-in-sql.md:137]:
```suggestion
| `SURVIVAL_PREFERENCE`      | 指定按 label 容灾等级的优先级放置副本。例如 `SURVIVAL_PREFERENCE="[region, zone, host]"`。    |
```

File: placement-rules-in-sql.md [hunk 9]
```diff
@@ -247,6 +248,25 @@ PARTITION BY RANGE( YEAR(purchased) ) (
 >
 > 字典和列表格式都基于 YAML 解析，但 YAML 语法有些时候不能被正常解析。例如 YAML 会把 `"{+disk=ssd:1,+disk=nvme:2}"`（`:` 后无空格）错误地解析成 `'{"+disk=ssd:1": null, "+disk=nvme:2": null}'`，不符合预期。但 `"{+disk=ssd: 1,+disk=nvme: 2}"`（`:` 后有空格）能被正确解析成 `'{"+disk=ssd": 1, "+disk=nvme": 2}'`。
 
+### 生存偏好
+
+一些重要的数据可能需要跨可用区存储多个副本，从而具备高的容灾生存能力，比如具备云区域级别的生存能力。在创建或修改放置策略时，你可以使用 `SURVIVAL_PREFERENCES` 选项设置数据的生存能力偏好。
```
Comment (hfxsd) [inline placement-rules-in-sql.md:253]:
```suggestion
一些重要的数据可能需要跨可用区存储多个副本，从而具备较高的容灾生存能力，比如具备云区域级别的生存能力。在创建或修改放置策略时，你可以使用 `SURVIVAL_PREFERENCES` 选项设置数据的生存能力偏好。
```

**Review by hfxsd — APPROVED** [review]
Rest LGTM

File: placement-rules-in-sql.md [hunk 10]
```diff
@@ -247,6 +248,25 @@ PARTITION BY RANGE( YEAR(purchased) ) (
 >
 > 字典和列表格式都基于 YAML 解析，但 YAML 语法有些时候不能被正常解析。例如 YAML 会把 `"{+disk=ssd:1,+disk=nvme:2}"`（`:` 后无空格）错误地解析成 `'{"+disk=ssd:1": null, "+disk=nvme:2": null}'`，不符合预期。但 `"{+disk=ssd: 1,+disk=nvme: 2}"`（`:` 后有空格）能被正确解析成 `'{"+disk=ssd": 1, "+disk=nvme": 2}'`。
 
+### 生存偏好
+
+一些重要的数据可能需要跨可用区存储多个副本，从而具备较高的容灾生存能力，比如具备云区域级别的生存能力。在创建或修改放置策略时，你可以使用 `SURVIVAL_PREFERENCES` 选项设置数据的生存能力偏好。
+
```
Comment (SunRunAway) [inline placement-rules-in-sql.md:254]:
```suggestion
#### 场景1 ：跨云区域的部署的数据库
```

File: placement-rules-in-sql.md [hunk 11]
```diff
@@ -247,6 +248,25 @@ PARTITION BY RANGE( YEAR(purchased) ) (
...
+例如，在创建放置策略时，你可以按照以下方式设置 `SURVIVAL_PREFERENCES`，要求数据尽量满足所设置的生存偏好：
+
+``` sql
+CREATE PLACEMENT POLICY multiregion
+    follower=4
+    PRIMARY_REGION="us-east-1"
+    SURVIVAL_PREFERENCES="[region, zone]";
+```
```
Comment (SunRunAway) [inline placement-rules-in-sql.md:262]:
```suggestion
``` sql
CREATE PLACEMENT POLICY multiaz SURVIVAL_PREFERENCES="[zone, host]";
CREATE PLACEMENT POLICY singleaz CONSTRAINTS="[+zone=zone1] SURVIVAL_PREFERENCES="[host]"
```

对于以上例子，应用了 `multiaz` 的数据，会以 3 副本的形式放置在不同的可用区里；应用了 `singleaz` 的数据，会以 3 副本的形式放置在名为 `zone1` 的单个可用区里。
```

File: placement-rules-in-sql.md [hunk 12]
```diff
@@ -247,6 +248,25 @@ PARTITION BY RANGE( YEAR(purchased) ) (
 >
 > 字典和列表格式都基于 YAML 解析，但 YAML 语法有些时候不能被正常解析。例如 YAML 会把 `"{+disk=ssd:1,+disk=nvme:2}"`（`:` 后无空格）错误地解析成 `'{"+disk=ssd:1": null, "+disk=nvme:2": null}'`，不符合预期。但 `"{+disk=ssd: 1,+disk=nvme: 2}"`（`:` 后有空格）能被正确解析成 `'{"+disk=ssd": 1, "+disk=nvme": 2}'`。
 
+### 生存偏好
+
+一些重要的数据可能需要跨可用区存储多个副本，从而具备较高的容灾生存能力，比如具备云区域级别的生存能力。在创建或修改放置策略时，你可以使用 `SURVIVAL_PREFERENCES` 选项设置数据的生存能力偏好。
```
Comment (SunRunAway) [inline placement-rules-in-sql.md:253]:
```suggestion
你可以使用 `SURVIVAL_PREFERENCES` 选项设置数据的生存能力偏好。
```

**Review by SunRunAway — APPROVED** [review]

File: placement-rules-in-sql.md [hunk 13]
```diff
@@ -247,6 +248,23 @@ PARTITION BY RANGE( YEAR(purchased) ) (
 >
 > 字典和列表格式都基于 YAML 解析，但 YAML 语法有些时候不能被正常解析。例如 YAML 会把 `"{+disk=ssd:1,+disk=nvme:2}"`（`:` 后无空格）错误地解析成 `'{"+disk=ssd:1": null, "+disk=nvme:2": null}'`，不符合预期。但 `"{+disk=ssd: 1,+disk=nvme: 2}"`（`:` 后有空格）能被正确解析成 `'{"+disk=ssd": 1, "+disk=nvme": 2}'`。
 
+### 生存偏好
+
+在创建或修改放置策略时，你可以使用 `SURVIVAL_PREFERENCES` 选项设置数据的生存能力偏好。
+
+例如，在一个部署在 3 个可用区，每个可用区上的节点可能混布多个进程的拓扑的集群:
```
Comment (qiancai) [inline placement-rules-in-sql.md]:
```suggestion
例如，对于一个分布在 3 个可用区的集群，如果每个可用区上的节点混合部署了多个进程的拓扑，在创建放置策略时可以按照以下方式设置 `SURVIVAL_PREFERENCES`：
```

File: placement-rules-in-sql.md [hunk 14]
```diff
@@ -247,6 +248,23 @@ PARTITION BY RANGE( YEAR(purchased) ) (
 >
 > 字典和列表格式都基于 YAML 解析，但 YAML 语法有些时候不能被正常解析。例如 YAML 会把 `"{+disk=ssd:1,+disk=nvme:2}"`（`:` 后无空格）错误地解析成 `'{"+disk=ssd:1": null, "+disk=nvme:2": null}'`，不符合预期。但 `"{+disk=ssd: 1,+disk=nvme: 2}"`（`:` 后有空格）能被正确解析成 `'{"+disk=ssd": 1, "+disk=nvme": 2}'`。
 
+### 生存偏好
+
+在创建或修改放置策略时，你可以使用 `SURVIVAL_PREFERENCES` 选项设置数据的生存能力偏好。
+
+例如，在一个拓扑是部署在 3 个 `zone` (即可用区），每个可用区的 `host`（即节点）上混合部署多个 tikv 实例的集群上:
```
Comment (qiancai) [inline placement-rules-in-sql.md:255]:
```suggestion
例如，假设你有一个集群，它分布在 3 个 `zone` (即可用区），且每个可用区的 `host`（即节点）上混合部署了多个 TiKV 实例。在为该集群创建放置策略时，可以参考以下方式设置 `SURVIVAL_PREFERENCES`：
```

File: placement-rules-in-sql.md [hunk 15]
```diff
@@ -247,6 +248,25 @@ PARTITION BY RANGE( YEAR(purchased) ) (
...
+
+例如，假设你有一个集群，它分布在 3 个 `zone` (即可用区），且每个可用区的 `host`（即节点）上混合部署了多个 TiKV 实例。在为该集群创建放置策略时，按如下方式设置上 `SURVIVAL_PREFERENCES`：
+
+``` sql
+CREATE PLACEMENT POLICY multiaz SURVIVAL_PREFERENCES="[zone, host]";
+CREATE PLACEMENT POLICY singleaz CONSTRAINTS="[+zone=zone1]" SURVIVAL_PREFERENCES="[host]";
+```
+
+创建好放置策略后，你可以按需将放置策略绑定到对应的表上：
+- 对于绑定了 `multiaz` 放置策略的表，数据将以 3 副本的形式放置在不同的可用区里，优先满足跨 `zone`（即可用区）级别数据隔离的生存目标，再满足跨 `host`（即机器）级别的数据隔离的生存目标。
```
Comment (qiancai) [inline placement-rules-in-sql.md:263]:
```suggestion
- 对于绑定了 `multiaz` 放置策略的表，数据将以 3 副本的形式放置在不同的可用区里，优先满足跨 `zone`（即可用区）级别数据隔离的生存目标，再满足跨 `host`（即节点）级别的数据隔离的生存目标。
```

File: placement-rules-in-sql.md [hunk 16]
```diff
@@ -247,6 +248,25 @@ PARTITION BY RANGE( YEAR(purchased) ) (
...
+- 对于绑定了 `singleaz` 放置策略的表，数据会优先以 3 副本的形式全部放置在 `zone1` 这个可用区里，再满足跨 `host`（即机器）级别的数据隔离的生存目标。
```
Comment (qiancai) [inline placement-rules-in-sql.md:264]:
```suggestion
- 对于绑定了 `singleaz` 放置策略的表，数据会优先以 3 副本的形式全部放置在 `zone1` 这个可用区里，再满足跨 `host`（即节点）级别的数据隔离的生存目标。
```

File: placement-rules-in-sql.md [hunk 17]
```diff
@@ -247,6 +248,25 @@ PARTITION BY RANGE( YEAR(purchased) ) (
 >
 > 字典和列表格式都基于 YAML 解析，但 YAML 语法有些时候不能被正常解析。例如 YAML 会把 `"{+disk=ssd:1,+disk=nvme:2}"`（`:` 后无空格）错误地解析成 `'{"+disk=ssd:1": null, "+disk=nvme:2": null}'`，不符合预期。但 `"{+disk=ssd: 1,+disk=nvme: 2}"`（`:` 后有空格）能被正确解析成 `'{"+disk=ssd": 1, "+disk=nvme": 2}'`。
 
+### 生存偏好
+
+在创建或修改放置策略时，你可以使用 `SURVIVAL_PREFERENCES` 选项设置数据的生存能力偏好。
+
+例如，假设你有一个集群，它分布在 3 个 `zone` (即可用区），且每个可用区的 `host`（即节点）上混合部署了多个 TiKV 实例。在为该集群创建放置策略时，按如下方式设置上 `SURVIVAL_PREFERENCES`：
```
Comment (qiancai) [inline placement-rules-in-sql.md:255]:
```suggestion
例如，假设你有一个 TiDB 集群，它分布在 3 个 `zone` (即可用区），且每个可用区的 `host`（即节点）上混合部署了多个 TiKV 实例。在为该集群创建放置策略时，假设 `SURVIVAL_PREFERENCES` 的设置如下：
```

File: placement-rules-in-sql.md [hunk 18]
```diff
@@ -247,6 +248,25 @@ PARTITION BY RANGE( YEAR(purchased) ) (
...
+CREATE PLACEMENT POLICY singleaz CONSTRAINTS="[+zone=zone1]" SURVIVAL_PREFERENCES="[host]";
+```
+
+创建好放置策略后，你可以按需将放置策略绑定到对应的表上：
+- 对于绑定了 `multiaz` 放置策略的表，数据将以 3 副本的形式放置在不同的可用区里，优先满足跨 `zone`（即可用区）级别数据隔离的生存目标，再满足跨 `host`（即节点）级别的数据隔离的生存目标。
+- 对于绑定了 `singleaz` 放置策略的表，数据会优先以 3 副本的形式全部放置在 `zone1` 这个可用区里，再满足跨 `host`（即节点）级别的数据隔离的生存目标。
+
+> **注意：**
+>
+> `SURVIVAL_PREFERENCES` 和 PD 中的 `LOCATION_LABELS` 是等价的，更多信息可以参考[通过拓扑 label 进行副本调度](/schedule-replicas-by-topology-labels.md)。
```
Comment (qiancai) [inline placement-rules-in-sql.md:268]:
```suggestion
> `SURVIVAL_PREFERENCES` 和 PD 中的 `location-labels` 是等价的，更多信息可以参考[通过拓扑 label 进行副本调度](/schedule-replicas-by-topology-labels.md)。
```

File: placement-rules-in-sql.md [hunk 19]
```diff
@@ -247,6 +248,25 @@ PARTITION BY RANGE( YEAR(purchased) ) (
 >
 > 字典和列表格式都基于 YAML 解析，但 YAML 语法有些时候不能被正常解析。例如 YAML 会把 `"{+disk=ssd:1,+disk=nvme:2}"`（`:` 后无空格）错误地解析成 `'{"+disk=ssd:1": null, "+disk=nvme:2": null}'`，不符合预期。但 `"{+disk=ssd: 1,+disk=nvme: 2}"`（`:` 后有空格）能被正确解析成 `'{"+disk=ssd": 1, "+disk=nvme": 2}'`。
 
+### 生存偏好
+
+在创建或修改放置策略时，你可以使用 `SURVIVAL_PREFERENCES` 选项设置数据的生存能力偏好。
+
+例如，假设你有一个 TiDB 集群，它分布在 3 个 `zone` (即可用区），且每个可用区的 `host`（即节点）上混合部署了多个 TiKV 实例。在为该集群创建放置策略时，假设 `SURVIVAL_PREFERENCES` 的设置如下：
```
Comment (hfxsd) [inline placement-rules-in-sql.md:255]:
```suggestion
例如，假设你的 TiDB 集群分布在 3 个 `zone`（即可用区），且每个可用区的 `host`（即节点）上混合部署了多个 TiKV 实例。在为该集群创建放置策略时，假设 `SURVIVAL_PREFERENCES` 的设置如下：
```

File: placement-rules-in-sql.md [hunk 20]
```diff
@@ -247,6 +248,25 @@ PARTITION BY RANGE( YEAR(purchased) ) (
...
+
+``` sql
+CREATE PLACEMENT POLICY multiaz SURVIVAL_PREFERENCES="[zone, host]";
+CREATE PLACEMENT POLICY singleaz CONSTRAINTS="[+zone=zone1]" SURVIVAL_PREFERENCES="[host]";
+```
+
+创建好放置策略后，你可以按需将放置策略绑定到对应的表上：
+- 对于绑定了 `multiaz` 放置策略的表，数据将以 3 副本的形式放置在不同的可用区里，优先满足跨 `zone`（即可用区）级别数据隔离的生存目标，再满足跨 `host`（即节点）级别的数据隔离的生存目标。
+- 对于绑定了 `singleaz` 放置策略的表，数据会优先以 3 副本的形式全部放置在 `zone1` 这个可用区里，再满足跨 `host`（即节点）级别的数据隔离的生存目标。
```
Comment (hfxsd) [inline placement-rules-in-sql.md:264]:
```suggestion
- 对于绑定了 `multiaz` 放置策略的表，数据将以 3 副本的形式放置在不同的可用区里，优先满足跨 `zone` 级别数据隔离的生存目标，再满足跨 `host` 级别的数据隔离的生存目标。
- 对于绑定了 `singleaz` 放置策略的表，数据会优先以 3 副本的形式全部放置在 `zone1` 这个可用区里，再满足跨 `host` 级别的数据隔离的生存目标。
```

**Review by qiancai — APPROVED** [review]
LGTM

File: placement-rules-in-sql.md [hunk 21]
```diff
@@ -247,6 +248,25 @@ PARTITION BY RANGE( YEAR(purchased) ) (
...
+在创建或修改放置策略时，你可以使用 `SURVIVAL_PREFERENCES` 选项设置数据的生存能力偏好。
+
+例如，假设你的 TiDB 集群分布在 3 个 `zone`（即可用区），且每个可用区的 `host`（即节点）上混合部署了多个 TiKV 实例。在为该集群创建放置策略时，假设 `SURVIVAL_PREFERENCES` 的设置如下：
+
+``` sql
+CREATE PLACEMENT POLICY multiaz SURVIVAL_PREFERENCES="[zone, host]";
+CREATE PLACEMENT POLICY singleaz CONSTRAINTS="[+zone=zone1]" SURVIVAL_PREFERENCES="[host]";
+```
+
+创建好放置策略后，你可以按需将放置策略绑定到对应的表上：
```
Comment (qiancai) [inline placement-rules-in-sql.md:262]:
```suggestion
创建好放置策略后，你可以按需将放置策略绑定到对应的表上：

```

**Pull request merged**